searchlib/src/vespa/searchlib/engine/trace.cpp

```diff
@@ -59,12 +80,14 @@ Trace::addTimeStamp(Cursor & trace) {
 }
 
 void Trace::done() {
-    _root.setDouble("duration_ms", _relativeTime.timeSinceDawn()/1000000.0);
+    if (!_root) { return; }
+
+    _root->setDouble("duration_ms", _relativeTime.timeSinceDawn()/1000000.0);
```

Comment:
should not access root directly, could use has_root predicate to avoid creating root.

Reply:
Fixed as discussed